wrapper_test.go

```diff
@@ -0,0 +1,46 @@
+package nmt
+
+import (
+	"testing"
+
+	"github.com/lazyledger/rsmt2d"
+)
+
+const (
+	shareSize           = 256
+	adjustedMessageSize = shareSize - DefaultNamespaceIDLen
+)
+
+func TestPushErasuredNamespacedMerkleTree(t *testing.T) {
```

Comment:
I wasn't sure what else I could test, so any suggestions are greatly appreciated.

Reply:
I think there is not much to test as the wrapper is really simple. And that is good. Well done!

The only thing I'd suggest is to also cover the non-happy paths in unit tests (e.g. the few edge cases where this can panic/error).